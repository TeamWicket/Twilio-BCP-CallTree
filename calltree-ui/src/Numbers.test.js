import { NumberList, NumberCreate, NumberEdit } from './Numbers';
import React from 'react';
import { shallow } from 'enzyme';

describe("Number Component", () => {
    it("should match snapshot of Number List Component", () => {
        expect(shallow(<NumberList />)).toMatchSnapshot();
    })
    it("should match snapshot of Number Create Component", () => {
        expect(shallow(<NumberCreate />)).toMatchSnapshot();
    })
    it("should match snapshot of NumberEdit Component", () => {
        expect(shallow(<NumberEdit />)).toMatchSnapshot();
    })
})
