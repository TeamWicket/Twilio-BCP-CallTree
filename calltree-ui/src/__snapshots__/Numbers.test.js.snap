// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Number Component should match snapshot of Number Create Component 1`] = `ShallowWrapper {}`;

exports[`Number Component should match snapshot of Number List Component 1`] = `ShallowWrapper {}`;

exports[`Number Component should match snapshot of NumberEdit Component 1`] = `ShallowWrapper {}`;
